package.json

```diff
@@ -36,13 +36,14 @@
   ],
   "scripts": {
     "changelog": "conventional-changelog -p angular -u",
-    "prebuild": "node ./scripts/generate-icon-exports.js && node ./scripts/generate-custom-properties.js",
-    "prestorybook:start": "node ./scripts/generate-icon-exports.js && node ./scripts/generate-custom-properties.js",
+    "prestorybook:build": "yarn run prebuild && node ./scripts/copy-svg-icons-to-assets.js",
```

Comment:
I'm confused as to why this needs to be associated with storybook? Seems more appropriate to do it as a prebuild step.

Reply:
Don't worry much about this stuff, it was to just to quickly get it to work